wow/core/src/main/scala/wow/auth/data/Account.scala

```diff
@@ -63,35 +59,18 @@ object Account extends SQLSyntaxSupport[Account] with RichColumn[Account] {
       .eq(c.id, account.id)
   }.update.apply()
 
-  def saveIdentity(login: String, identity: Srp6Identity)(implicit session: DBSession = autoSession): Unit = assert(
-    withSQL {
-      update(Account)
-        .set(
-          c.salt -> identity.salt,
-          c.verifier -> identity.verifier
-        )
-        .where
-        .eq(c.login, login)
-    }.update.apply() > 0)
-
-  def saveSessionKey(login: String, sessionKey: BigInt)(implicit session: DBSession = autoSession): Unit = assert(
-    withSQL {
-      update(Account)
-        .set(
-          c.sessionKey -> sessionKey
-        )
-        .where
-        .eq(c.login, login)
-    }.update.apply() > 0)
-
-  def create(login: String, identity: Srp6Identity)(implicit session: DBSession = autoSession): Long = withSQL {
-    insert.into(Account)
-      .namedValues(
-        c.login -> login.toUpperCase(),
-        c.salt -> identity.salt,
-        c.verifier -> identity.verifier
-      )
-  }.updateAndReturnGeneratedKey.apply()
+  def find(condition: SQLSyntax)(implicit session: DBSession = autoSession): Option[Account] = withSQL {
```

Comment:
It is not. While it might seem weird to expose a public method using taking an SQLSyntax parameter, the goal was to keep the Account object extensible by allowing search based on any criteria.

Since that method is going to be exist anyway as a way to avoid code redundancy between the other two search functions, I thought it might as well be public.